spec/unit/settings/config_file_spec.rb

```diff
@@ -123,7 +123,7 @@ def the_parse_of(*lines)
 
     expect { config.parse_file(filename, text, [:legal]) }.
       to raise_error Puppet::Error,
-        /Illegal section 'legal' in config file #{filename} at line 1/
+        /Illegal section 'legal' in config file at \(file: #{filename}, line: 1\)/
```

Comment:
Did this one not need your `_()` string decorations for translation?

Reply:
This is test rspec test code. S, no it does not need to be translated